stdlib/src/bit/bit.mojo

```diff
@@ -253,6 +253,23 @@ fn bit_not[
 # ===----------------------------------------------------------------------===#
 
 
+@always_inline
+fn bit_width(val: Int) -> Int:
```

Comment:
Can you please add tests for this? Also, while you're at it, could you also add tests for the SIMD overload?